Dockerfile

```diff
@@ -11,7 +11,4 @@ WORKDIR /app
 COPY requirements.txt .
 RUN pip install -r /app/requirements.txt
 
-# With `docker compose`, dev. directory already mounted at /app
-#COPY . .
-
```

Comment:
If this was set up to run as a job using Jenkins/OpenShift, I think you'd need this, otherwise the source code isn't in the image (or container). It shouldn't be a problem to copy this over, I think most of our applications copy and then also mount the directory for local development.

Reply:
Thanks for mentioning that.  I'd forgotten about that.  I'll restore the lines.

Reply:
Restored!